assets/css/style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+    --paddi: 10%;
+}
+
+header {
+    height: 40vh;
+    border: 1px solid rgb(0, 0, 0);
+    display: flex;
+    flex-direction: column;
+    background-image: url("../images/cafe-blur-light.jpg");
+    background-size: 100% 100%;
+}
+
+.top-menu {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+}
+
+.menu-item {
+    width: 50%;
+}
+
+.menu-item:first-child {
+    padding-left: var(--paddi);
+}
+
+.menu-item a img {
+    width: 70px;
+    height: 60px;
+}
+
+.main-pages {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 30%;
+    margin-right: var(--paddi);
+    align-items:center;
+}
+
+.main-pages a {
+    padding: 0px 10px;
+    text-decoration: none;
+    color: black;
+    font-weight: bolder;
+    font-size: 1.2rem;
+}
+
+.main-pages a:hover {
+    color: brown;
+}
+
+.main-pages a:last-child {
+    padding: 0px 0px 0px 10px;
+}
+
+.small-circle {
+    height: 5px;
+    width: 5px;
+    background-color: rgb(0, 0, 0);
+    border-radius: 50%;
+}
+
+.description {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.main {
+    width: 100vh;
+}
+
+.bar {
+    border-top: 5px solid white;
+    border-bottom: 5px solid white;
+    padding: 10px 0;
+    display: flex;
+    justify-content: space-between;
+    background-color: #DFCDC9;
+}
+
+.subnav-bar {
+    padding: 0 0 0 var(--paddi);
+    width: 25%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.price-filter {
+    padding: 0 var(--paddi);
+    display: flex;
+}
+
+#site-main {
+    height: auto;
+}
+
+#menu {
+    width: 100%;
+    padding: 0px var(--paddi);
+    display: flex;
+    flex-wrap: wrap;
+    list-style-type: none;
+}
+
+#menu li {
+    border: 10px solid rgb(246, 246, 246);
+    justify-content: space-between;
+    width: 20%;
+}
+
+#menu li:hover {
+    border: 0px;
+    width: 21%;
+    cursor: pointer;
+}
+
+#menu li div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.icon{
+    color:rgb(0, 0, 0);
+    display: flex;
+    align-items: center;
+}
+
+/* TEXT conventions */
+#menu-text{
+    font-weight: bold;
+    color: black;
+}
+
+.title-text{
+    font-weight: bold; 
+    font-size: 40px;
+    color:rgb(0, 0, 0);
+}
+
+.heading-text{
+    font-weight: bold; 
+    font-size: 25px;
+    color:rgb(0, 0, 0);
+}
+
+.title-line{
+    width:50%;
+     height:5px;
+      color:"#DFCDC9";
+      background-color: #DFCDC9;
+}
+
+.dish-img img {
+    width: 80%;
+    height: 200px;
+}
+
+.subnav-content {
+    display: none;
+    position: absolute;
+    background-color: #DFCDC9;
+    min-width: 100px;
+}
+
+.subnav-menu:hover .subnav-content {
+    display: block;
+    line-height: 20px;
+    padding: 0 10px;
+    cursor: pointer;
+}
+
+.footer {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
It's good practice to have newlines at the end of files. There is a VSCode setting to do that automatically on save. Otherwise, you can see this icon on Pull Requests:
![image](https://github.com/andreeap00/assignment1/assets/19929642/9dc343b4-122b-4a49-880d-bbefd655bf22)